src/SchemaForm.vue

```diff
@@ -1,21 +1,24 @@
 <template>
-    <div class="schema-form">
-      <slot name="beforeForm"></slot>
-      <component
-        v-for="field in parsedSchema"
-        :key="field.model"
-        :is="field.component"
-        v-bind="binds(field)"
-        :modelValue="val(field)"
-        @update:modelValue="update(field.model, $event)"
-        @update-batch="updateBatch(field.model, $event)"
-      />
-      <slot name="afterForm"></slot>
-    </div>
+    <component
+      :is="!hasParentSchema ? 'form' : 'div'"
+      class="schema-form"
```

Comment:
What does the `class` do here?

Reply:
Its being used by parents to style, and i think users may want to use it to target this particular form in style sheets

Reply:
Hm. Given that `class` and `style` are being attached to the root element, do we really need it?

Reply:
Alright sounds good